### ISSUE TITLE
Allow subtitle accessibility overrides on Apple TV

### ISSUE THREAD

**Issue by gannon345** [jellyfin/Swiftfin#1726]
**Describe the feature you'd like**
Allow Swiftfin to override subtitle fonts to use Apple TV's built in accessibility features.

**Additional context**
Apple TV allows for built in accessibility features to define subtitle font, color, and style. Would like to have Swiftfin allow this to be used in place of the fonts included in content. Plex and some streaming services have similar behavior to allow this, but I've not seen this on other video clients, including Swiftfin.

Specifically, these would be to allow the subtitles from:

Apple TV Settings -> Accessibility -> Subtitles and Captioning -> Style

**Comment by JPKribs** [comment]
This exists on our Development branch for tvOS and should be included in our next release. You can follow our progress towards this release here:

https://github.com/jellyfin/Swiftfin/discussions/1294

If you're able, please test Swiftfin on your iPhone/iPad where this feature currently exists and let us know if these settings handle the accessibility requirements.
